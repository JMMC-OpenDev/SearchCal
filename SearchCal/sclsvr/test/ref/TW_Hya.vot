version https://git-lfs.github.com/spec/v1
oid sha256:81918a8b72b34b6d84ea0152337e35a05d069d5b6107414e7cde05f72d4dfc65
size 192665
