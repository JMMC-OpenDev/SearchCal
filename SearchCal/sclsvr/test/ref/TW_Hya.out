version https://git-lfs.github.com/spec/v1
oid sha256:49b5da2c9ab00bf222d9eee26d578b2c828054d1d42f4f68f4ca1a6c6a0c98db
size 296165
